lib/instance.js

```diff
@@ -321,8 +321,7 @@ module.exports = (function() {
               value = (value === 1) ? true : (value === 0) ? false : value;
             }
           }
-
-          if (!options.raw && (primitives.indexOf(typeof value) === -1 || value !== originalValue)) {
+          if (!options.raw && (primitives.indexOf(typeof value) === -1 && value !== null || value !== originalValue)) {
```

Comment:
I'd say it's easier to read it without the extra parentheses but it's not really important...